j-commands.html

```diff
@@ -0,0 +1,408 @@
+choice to use these as required.)</p>
+
+<p>This command is repeatable by passing a list of theorems.</p>
+
+<code>$j natded_true 'TRUE' with 'THEOREM' ...;</code>
+
+<p>Registers `THEOREM` as a natural deduction rule pertaining to
+the true constant `TRUE` (which does not necessarily have to coincide
+with the `TRUE` constant used for the empty context in `natded_init`).
+These have no specified form besides the use of `| (CONJ -&gt; A)` and
+`|- A` hypotheses and conclusion, but they will generally involve
+introducing true as in `|- T.` or `|- ( A -&gt; T. )` or eliminating
+true on the left as in `( T. -&gt; A ) |- A`. (Automation has free
+choice to make use of these lemmas as required, but an analogue
+of `tru` is recommended.)</p>
+
+<p>This command is repeatable by passing a list of theorems, or
+it can be split into multiple commands; the `'TRUE' with` part must
+be repeated in this case. Multiple `TRUE` constants can also be
+introduced by using multiple commands.</p>
+
+<code>$j natded_imp 'IMP' with 'THEOREM' ...;</code>
+
+<p>Registers `THEOREM` as a natural deduction rule pertaining to
+implication `IMP` (which does not necessarily have to coincide
+with the `IMP` constant used for the sequent operator in `natded_init`).
+These have no specified form besides the use of
+`| (CONJ -&gt; A)` and `|- A` hypotheses and conclusion, but they
+will generally involve introducing implication as in
+`(A /\ B) -&gt; C |- A -&gt; (B -&gt; C)` and eliminating it as in
+`A -&gt; B, A -&gt; (B -&gt; C) |- A -&gt; C`. (Automation has
+free choice to make use of these lemmas as required, but analogues
+of `ex` and `mpd` are recommended.)</p>
+
+<p>This command is repeatable by passing a list of theorems, or it
+can be split into multiple commands; the `'IMP' with` part must be
+repeated in this case. Multiple `IMP` constants can also be
+introduced by using multiple commands.</p>
+
+<code>$j natded_and 'AND' with 'THEOREM' ...;</code>
+
+<p>Registers `THEOREM` as a natural deduction rule pertaining to
+conjunction `AND` (which does not necessarily have to coincide
+with the `AND` constant used for the context conjunction in
+`natded_init`). These have no specified form besides the use
+of `| (CONJ -&gt; A)` and `|- A` hypotheses and conclusion, but they
+will generally involve introducing conjunction as in
+`A -&gt; B, A -&gt; C |- A -&gt; (B /\ C)` and eliminating it as in
+`A -&gt; (B /\ C) |- A -&gt; B`. (Automation has free choice to
+make use of these lemmas as required, but analogues of `jca`,
+`simpld` and `simprd` are recommended.)</p>
+
+<p>This command is repeatable by passing a list of theorems,
+or it can be split into multiple commands; the `'AND' with` part
+must be repeated in this case. Multiple `AND` constants can also be
+introduced by using multiple commands.</p>
+
+<code>$j natded_or 'OR' with 'THEOREM' ...;</code>
+
+<p>Registers `THEOREM` as a natural deduction rule pertaining to
```

Comment:
```suggestion
<p>Registers <code>THEOREM</code> as a natural deduction rule pertaining to
```
Backticks were used in the original post for code spans. A search/replace for " \`" and "\` " / "\`," should get most of them.

Reply:
Done.